adapters/huaweiads/huaweiads.go

```diff
@@ -547,19 +550,20 @@ func getNativeFormat(adslot30 *adslot30, openRTBImp *openrtb2.Imp) error {
 		if asset.Img != nil {
 			if asset.Img.Type == native1.ImageAssetTypeMain {
 				numMainImage++
-				if asset.Img.H != 0 && asset.Img.W != 0 {
+				/*if asset.Img.H != 0 && asset.Img.W != 0 {
 					width = asset.Img.W
 					height = asset.Img.H
 				} else if asset.Img.WMin != 0 && asset.Img.HMin != 0 {
 					width = asset.Img.WMin
 					height = asset.Img.HMin
-				}
+				}*/
```

Comment:
Same here. Please remove the commented out code.

Reply:
removed.